src/components/taskDetails/ProgressDetails.tsx

```diff
@@ -1,81 +0,0 @@
-import { useState, FC, MouseEvent } from 'react';
-import { ProgressDetailsData } from '@/types/standup.type';
-import { getDateFromTimestamp } from '@/utils/getDateFromTimestamp';
-import styles from './progress-details.module.scss';
-import { useRouter } from 'next/router';
-import ProgressUpdateCard from './ProgressUpdateCard/ProgressUpdateCard';
-
-type Props = {
-    data: ProgressDetailsData;
-};
-const ProgressDetails: FC<Props> = ({ data }) => {
-    const router = useRouter();
-    const [showProgressDetails, setShowProgressDetails] =
-        useState<boolean>(false);
-
-    const openDetails = (e: MouseEvent<HTMLElement>) => {
-        setShowProgressDetails(true);
-    };
-
-    const closeDetails = () => {
-        setShowProgressDetails(false);
-    };
-
-    const isDev = router.query.dev === 'true';
-    return (
-        <>
-            {isDev ? (
-                <ProgressUpdateCard data={data} />
-            ) : (
-                <li
-                    onClick={openDetails}
-                    className={styles['list-item']}
-                    data-testid="progress-item"
-                >
-                    {getDateFromTimestamp(data.date)}
-                </li>
-            )}
```

Comment:
- As we have changed from list to cards we won't need list element anymore. So it was removed. 
- Once we remove list element `openDetails` function becomes redundant and `showProgressDetails` state would never become true. So code from line no. 38  to 77 becomes redundant. 
- So now we are just left with `ProgressDetails` component just as a wrapper for `ProgressUpdateCard` component and ProgressDetails component is only used here : https://github.com/Real-Dev-Squad/website-status/blob/4ff1353a6db945a7a5d8d2a1d155ead65bd1b8ad/src/components/ProgressCard/index.tsx#L64
So instead of `ProgressDetails` i have used `ProgressUpdateCard` directly in `ProgressCard` component and deleted the `ProgressDetails`